### DISCUSSION TITLE
How to change the font name

### DISCUSSION THREAD

**Discussion by lednhatkhanh** [MonoLisaFont/feedback#176] [Q&A]
Just bought V2 and I cannot change the font name like V1 again, I added the suffix `Customed` and install the font to MacOS, it appears "MonoLisa Custom" in FontBook but I still need to use this exact font name for vscode to work. Please help, thank you.
<img width="246" alt="image" src="https://user-images.githubusercontent.com/9303093/206857142-5bc9cd04-7573-480a-95aa-6e4a2cc26644.png">

**Comment by okonet** [comment]
Have you used customize tool or did you try changing the name of the font with default settings?
